pkg/ebpf/c/common/namespaces.h

```diff
@@ -1,10 +1,11 @@
-#ifndef __TRACEE_NAMESPACES_H__
-#define __TRACEE_NAMESPACES_H__
+#ifndef __COMMON_NAMESPACES_H__
+#define __COMMON_NAMESPACES_H__
 
 #include <vmlinux.h>
+#include <vmlinux_flavors.h>
+#include <vmlinux_missing.h>
```

Comment:
I don't see a reason to include these headers here

Reply:
ditto (tried to create a pattern, can remove it).